### PR TITLE
Update hrv_OCRFixReplaceList.xml

### PR DESCRIPTION
**Pull request by diomed**

### CODE REVIEW

File: Dictionaries/hrv_OCRFixReplaceList.xml [hunk 1]
```diff
@@ -1358,7 +1326,9 @@
     <RegEx find="\b([vV])(išlj)(a|e|i|u|eg|em|oj|om)" replaceWith="$1iš$3" />
     <RegEx find="\b([vV])(odk)(a|e|i|u|ama)" replaceWith="$1otk$3" />
     <RegEx find="([vV])(ole)(n|na|no|ti|nu|nog|la|le|li)" replaceWith="$1olje$3" />
-    <RegEx find="([vV])(redi)(m|š|mo|te|li)?" replaceWith="$1rijedi$3" />
+    <RegEx find="\b([vV])(oz)(a|u|om)?\b" replaceWith="$1lak$3" />
```
Comment (ivandrofly) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: You don't need to `(oz)` and simply `replaceWith="$1lak$2"`

Comment (diomed) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: old habits die hard :stuck_out_tongue:

Comment (ivandrofly) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: :laughing:

**Comment by ivandrofly** [comment]
If you branch isn't merged you could run to avoid multiply commit with same comment
`git commit -a --amend` &  `git push --force HEAD:patch-1`

**Comment by ivandrofly** [comment]
There will be a merge conflict with https://github.com/SubtitleEdit/subtitleedit/pull/949/files I cloned this branch I'll merge it and PR later

**Comment by ivandrofly** [comment]
All done in: https://github.com/SubtitleEdit/subtitleedit/pull/948 @diomed  can be close

**Comment by ivandrofly** [comment]
:+1:

**Pull request closed without merging**